modules/veteran_verification/app/controllers/veteran_verification/metadata_controller.rb

```diff
@@ -0,0 +1,46 @@
+# frozen_string_literal: true
+
+require 'appeals_api/health_checker'
```

Comment:
Should this module rely on appeals?

Reply:
Nope!  Was a missed dependency removal when porting over the metadata controller.  Will get that removed.